source/semver.d

```diff
@@ -311,7 +311,7 @@ struct SemVer
                 return ids[i] < other.ids[i] ? -1 : 1;
         }
 
-        int compareSufix(const string[] suffix, const string[] anotherSuffix) @safe pure
+        int compareSufix(const scope string[] suffix, const string[] anotherSuffix) @safe pure
```

Comment:
Feels like `anotherSuffix` could be `scope` too ?
Also prefer `scope const` over `const scope`, as `const` is a type qualifier and `scope` is a storage class.

Reply:
Yes, reasonable.